packages/kotti-ui/source/kotti-card/KtCard.vue

```diff
@@ -27,19 +27,13 @@ export default defineComponent<KottiCard.PropsInternal>({
 	props: makeProps(KottiCard.propsSchema),
 	setup(props) {
 		return {
-			imageContainerClass: computed(() => ({
-				'kt-card__image-row': true,
-				'kt-card__image-row--is-bottom':
-					props.imgPosition === KottiCard.ImagePosition.BOTTOM,
-				'kt-card__image-row--is-middle':
-					props.imgPosition === KottiCard.ImagePosition.MIDDLE,
-				'kt-card__image-row--is-top':
-					props.imgPosition === KottiCard.ImagePosition.TOP,
+			cardClass: computed(() => ({
+				'kt-card': true,
+				[`kt-card--has-${props.imgPosition}-image`]: true,
```

Comment:
This line has a probably unintended consequence:
Since the default value is `top`, this line will cause the card to have a zero padding on top, see screenshot from the documentation

<img width="692" alt="Screenshot 2023-03-24 at 04 50 47" src="https://user-images.githubusercontent.com/16950410/227419970-59869346-64a6-46ed-ac3c-6a894dfefc6f.png">

Reply:
ah good point, same for bottom, I'll make the condition `hasImage` instead of true